src/Overview/components/ProductInfo.jsx

```diff
@@ -1,15 +1,25 @@
 import React from 'react';
 import StarRating from '/src/Widgets/StarRating.jsx'
+import {FaFacebookSquare} from 'react-icons/fa'
+import {FaTwitterSquare} from 'react-icons/fa'
+import {FaPinterest} from 'react-icons/fa'
```

Comment:
all of these could be combined in one line i.e. import {FaFacebookSquare, FaTwitterSquare, FaPinterest} from 'react-icons/fa';